frontend/src/pages/DashboardPage/DashboardPage.tsx

```diff
@@ -1,39 +1,75 @@
 import { Button } from "@/components/ui/button";
 import { useNavigate } from "react-router-dom";
+import { useEffect, useState } from "react";
 import { DISCOVER_PATH } from "@/lib/paths";
 import StreamerCard from "@/components/StreamerCard/StreamerCard";
+import { fetchStreamerCardInfo } from '@/lib/utils';
+import { useAuthContext } from "@/Context/AuthProvider";
 
 export default function DashboardPage() {
   const navigate = useNavigate();
+  const { session } = useAuthContext();
+  const [userProfile, setUserProfile] = useState(null);
+  const [loading, setLoading] = useState(true);
 
-  //keep in mind in future below discover streamers button wont be positioned absolutely but rather be in a flex box container that also contains prompt of day and recent activity
-  return (
-    
-    <div className = "flex flex-row gap-1 p-14 min-h-screen">
+  useEffect(() => {
+
+    async function populateStreamerCard(){
+      if (!session || !session.access_token) {
+        console.error("No valid session");
+        //if there isnt a session we set loading to false because we currently arent checking profile
+        setLoading(false); 
+        return;
+      }
+
+      //once there is a session setLoading to true so we can display loading text since we are checking profile
+      try {
+        setLoading(true);
 
-      <div className = "flex-1">
-        <StreamerCard />
+        //fetch the information through fetchStreamerCardInfo function which calls our api and it returns a profile object
+        const profile = await fetchStreamerCardInfo(session?.access_token);
+        setUserProfile(profile)
+      } catch (err) {
+        console.error(`Failed to load profile, ${err}`);
+        //once we are done going through whole try catch block it means loading is done so set state back to false so that the card can be rendered
+      } finally {
+        setLoading(false);
+      }
+    }
+
+    populateStreamerCard();
+
+  }, [session]);
+  //placing session in dependency array because on page mount session might not be loaded yet so have this here so we can run the fetch user profile again if session gets updated
+
+  if(loading){
+    return (
+      <div className="flex items-center justify-center min-h-screen">
+        <div className="text-white">Loading your profile...</div>
       </div>
+    );  
+  }
 
-      <div className = "flex flex-col gap-6 flex-1 items-center">
+  //keep in mind in future below discover streamers button wont be positioned absolutely but rather be in a flex box container that also contains prompt of day and recent activity
```

Comment:
You can already do that, just use the flexbox with column-reverse as flex-direction for now. That way the button will be at the bottom of the column